docs/en/sql-reference/functions/url-functions.md

```diff
@@ -500,3 +500,39 @@ Result:
 │ http://bigmir.net/?c=d&e=f#g │ http://bigmir.net/?a=b#g │
 └──────────────────────────────┴──────────────────────────┘
 ```
+
+### resolveRelativeURL(RELATIVE_URL, BASE_URL)
+
+The resolveRelativeURL function resolves a relative URL against a base URL and returns the absolute URL. 
```

Comment:
A bit more context why it may be required

```suggestion
The `resolveRelativeURL` function resolves a relative URL to its absolute form based on a given base URL.
This is particularly useful to resolve relative URLs from HTML pages.

```